source/command-monitoring/tests/find.json

```diff
@@ -270,6 +270,135 @@
     },
     {
       "description": "A successful find event with a getmore and killcursors",
+      "ignore_if_server_version_greater_than": 3.0,
```

Comment:
It's different enough in that it would force a kill cursors on < 3.2 where the other would not. I don't think it hurts to be there.